software/DocCreator/src/GenerateDocument/Assistant.hpp

```diff
@@ -390,6 +390,7 @@ protected slots:
   bool _Shadow_shadEnable;
 
   bool _Phantom_phantEnable;
+  QString _PhantomPatternsPath = "../share/DocCreator/data/Image/phantomPatterns/";
```

Comment:
It is better to use the original code:
```suggestion
  QString _PhantomPatternsPath = Context::BackgroundContext::instance()->getPath() +
         "../Image/phantomPatterns/"; //B:TODO:UGLY !
```
Check if it works correctly first.